chromium/rules.js

```diff
@@ -340,6 +350,70 @@ RuleSets.prototype = {
     util.log(util.INFO, 'done removing rule');
     return true;
   },
+  
+  /**
+  * Load stored user rules
```

Comment:
This comment makes figuring out how this function is different from `loadStoredUserRules`. Can you make the distinction between "`get`" and "`load`"  more clear for these `StoredUserRules` functions.